docs/tutorial.md

```diff
@@ -0,0 +1,436 @@
+
+```shell
+twarc2 search "(echidna OR echidna's OR echidnas) -is:retweet" echidna.json
+
+twarc2 search "(platypus OR platpus's OR platypi OR platypusses OR platypuses) -is:retweet" platypus.json
+```
+
+Running these two commands will save the tweets matching each of those searches to two files on our disk, which we will use for the next sessions. While saving the
+
+![Screenshot showing the progress of the tweets being downloaded](images/twarc_progress_download.png)
+
+TIP: if you're not sure where the files above have been saved, you can run the command `cd` on Windows, or `pwd` on Mac to have your shell print out the folder in the filesystem where twarc has been working.
+
+## Understanding and transforming twitter JSON data
+
+Now that we've collected some data, it's time to take a look at it. Let's start by viewing the collected data in it's plainest form: as a text file. Although we named the file with an extension of `.json`, this is just a convention: the actual file content is a plain text in the [JSON](https://en.wikipedia.org/wiki/JSON) format. Let's open this file with our inbuilt text editor (notepad on Windows, ... on Mac).
+
+![Screenshot of the json file in notepad](images/json_echidna.png)
+
+You'll notice immediately that there is a *lot* of data in that file: tweets are rich objects, and we mentioned that twarc by default captures as much information as Twitter makes available. Further, the Twitter API provides data in a format that makes it convenient for machines to work with, but not so much for humans.
+
+## Making a CSV file from our collected tweets
+
+We don't recommend trying to manually parse this raw data unless you have specific needs that aren't covered by existing tools. So we're going to use the `twarc-csv` package that we installed earlier to do the heavy lifting of transforming the collected JSON into a more friendly comma-separated value ([CSV](https://en.wikipedia.org/wiki/Comma-separated_values)) file. CSV is a simple plaintext format, but unlike JSON format is easy to import or open with a spreadsheet.
+
+The `twarc-csv` package lets us use a `csv` command to transform the files from twarc:
+
+```shell
+twarc2 csv echidna.json echidna.csv
+
+twarc2 csv platypus.json platypus.csv
+```
+
+If we look at these files in our text editor again, we'll see a nice structure of one line per tweet, with all of the many columns for that tweet.
+
+![Screenshot of the plaintext CSV file in notepad](images/echidna_csv.png)
+
+Since we're going to do more analysis with the Pandas library to answer our question, we will want to create the CSV with only the columns of interest. This will reduce the time and amount of RAM you need to load your dataset. For example, the following commands produce CSV files with a small number of fields:
```

Comment:
```suggestion
Since we're going to do more analysis with the Pandas library to answer our question, we will want to create the CSV with only the columns of interest. This will reduce the time and amount of computer memory/RAM you need to load your dataset. For example, the following commands produce CSV files with a small number of fields:
```